yat-docs/docs/partner-guides/1_crypto_wallets.md

```diff
@@ -0,0 +1,304 @@
+```
+
+
+### Redirection of the User from Yat Web Flow to the Partner Application
+
+The user can choose to purchase a new Yat on the Yat web application. After the successful
+completion of this flow, the Yat web application calls the return URL previously defined by the partner application development team.
+
+**Return URL:** To be agreed upon by the Yat and partner development teams in pre-integration setup.
+
+** Query Parameters: **
+
+`eid:` The Yat that was purchased, if any.
+
+`refresh_token:` The same refresh token that was sent to the web application by the partner application in [Redirection of the User to the Yat Web Application](#redirection-of-the-user-to-the-yat-web-application).
+
+The partner application receives the query parameters as a result of the redirection from the Yat web application to the partner application, and the integration flow is completed.
+
+## 2. Connect existing Yat(s)
+
+Users who already own one or more Yats can connect their Yat(s) with their wallet. We recommend that partner apps enable the user to enter the journey from:
+
+* Push notification announcing the partnership to all users
+* Splash screens on app launch
+* Menu item within the app’s Settings or Profile section
+
+### 2A. Connect existing Yat(s) from Onboarding
+
+<a href="/img/flow2a-lrg.png" target="_blank"><img src="/img/flow2a-sml.png"/></a>
+
+### 2B. Connect existing Yat(s) from Settings
+
+<a href="/img/flow2b-lrg.png" target="_blank"><img src="/img/flow2b-sml.png"/></a>
+
+## 3. Manage connected Yats
+
+Partner application’s users can access the Yat dashboard directly from the partner app with the Manage Yats flow. The Yat Dashboard enables users to visualize, mint, transfer their Yat and more.
+
+<a href="/img/flow3-lrg.png" target="_blank"><img src="/img/flow3-sml.png"/></a>
+
+## 4. Send payment to a Yat
+
+If the user has connected an address with their Yat, addresses are set, any one can send crypto payments and the user will receive it in the correct address. Alice can send BTC to Bob’s Yat (🚀👽) without ever having to paste Bob’s P2PKH address. To top it all off, the partner app can even render a unique visualization associated with 🚀👽 created by Bob himself. Yat supports the top 100 cryptocurrencies as native tags. Additional currencies can be specified with the generic crypto tag.
+
+<a href="/img/flow4a-lrg.png" target="_blank"><img src="/img/flow4a-sml.png"/></a>
+
```

Comment:
@Krakaw 

need to add relevant endpoints to cover:
- Visualization - low-res webm file
- Visualization fallback if no viz exists
- Yat validation (lookup?)

Also add note on
- Error handling (if Yat has been transferred / minted and sold on OpenSea / etc.)
- How we handle the edge case of many:one for e.g. multiple ETH addresses associated with 1 yat

Reply:
I'll pull the latest Swagger Spec so those are included in the API reference